docs/definition-of-ready.md

```diff
@@ -1,24 +1,65 @@
-- reading order when there are differences between what is visible and what is read out loud
-- hidden text for screen readers, where necessary
-- colours and colour contrast
-- if something is tabular data or not and suggest HTML solutions or patterns
-- the right design patterns to provide the user with consistent line spacing and whitespace
-- non-breaking spaces and hyphens to stop unwanted word wrapping
-- simple, easy to understand content that meets the content style guide
-- how will this look on mobile (remember a zoomed in desktop screen will effectively become a mobile viewport)
-- how this will work without javascript
-- if the journey is more than a few screens and whether we allow the user to save and come back later
-- whether people using assistive technology might need extra time to complete tasks
+## Be inclusive
+
+To make sure accessibility is part of the story, include elements of the following:
+
+- [Definition of Ready](#definition-of-ready)
+  - [Be inclusive](#be-inclusive)
+    - [Established patterns](#established-patterns)
+    - [Service design](#service-design)
+    - [Content design](#content-design)
+    - [Visual design](#visual-design)
+    - [Design for assistive technology](#design-for-assistive-technology)
+    - [User research](#user-research)
+
+Most of this should be part of standard components but may help if part of a ticket.
+
+### Established patterns
+
+Unless you have good reasons, which you can back up with data and research, and you plan to upstream your new pattern into the Design System, then you should stick to the established patterns with your designs.
+
+You might question if something is tabular data or not and suggest different HTML solutions or patterns. Use the right design patterns to provide the user with consistent line spacing and whitespace.
+
+You can find the established patterns here:
+
+- [GOV.UK Design System](https://design-system.service.gov.uk/)
+- [HMRC Design Patterns](https://design.tax.service.gov.uk/hmrc-design-patterns/)
+
+### Service design
+
+- URLs should be human-readable
+- your service should work without JavaScript
+
+### Content design
+
+- write simple, easy to understand content that meets the [content style guide](https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style)
+- reading order should make sense when there are differences between what is visible and what is read out loud
+- write page titles that are the same as the `h1` heading (but be aware of the [personally identifiable information](https://design.tax.service.gov.uk/hmrc-design-patterns/page-title/#personally-identifiable-information) caveat)
+
+### Visual design
+
+- colour contrast ratios should be fully accessible
+- use non-breaking spaces and hyphens to stop unwanted word wrapping
```

Comment:
Hyphens is probably not the right choice here. Adding hyphenation support for when words wrap?